02-overview-surveys.Rmd

```diff
@@ -10,25 +10,25 @@ To gain a deeper understanding of survey design and implementation, there are ma
 
-Generally, survey researchers consider there to be seven main sources of error that fall into two major categories of Representation and Measurement [@groves2009survey]:
+Generally, survey researchers consider there to be seven main sources of error that fall under either Representation and Measurement [@groves2009survey]:
 <!--TODO: Check that we always refer to it as population of interest and not target population-->
 
 - Representation
-    - **Sampling Error**: Error produced when selecting a *sample*, the subset of the population, from the *sampling frame*, the list from which the sample is drawn (there is no sampling error if conducting a census)
     - **Coverage Error**: A mismatch between the *population of interest* (also known as the target population or study population) and the sampling frame
+    - **Sampling Error**: Error produced when selecting a *sample*, the subset of the population, from the *sampling frame*, the list from which the sample is drawn (there is no sampling error if conducting a census)
     - **Nonresponse Error**: Differences between those who responded and did not respond to the survey (unit nonresponse) or a given question (item nonresponse)
     - **Adjustment Error**: Error introduced during post-survey statistical adjustments
 - Measurement
     - **Validity**: A mismatch between the topic of interest and the question(s) used to collect that information
     - **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
     - **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjustments to data based on illogical responses)
 
-Almost every survey will have some errors. Researchers attempt to conduct a survey that reduces the *total survey error*, or the accumulation of all errors that may arise throughout the survey life cycle. By assessing these different types of errors together, researchers can seek strategies to maximize the overall survey quality and improve the reliability and validity of results [@tse-doc]. However, attempts to lower individual sources errors (and therefore total survey error) come at the price of time, resources, and money:
+Almost every survey will have some errors. Researchers attempt to conduct a survey that reduces the *total survey error*, or the accumulation of all errors that may arise throughout the survey life cycle. By assessing these different types of errors together, researchers can seek strategies to maximize the overall survey quality and improve the reliability and validity of results [@tse-doc]. However, attempts to lower individual sources errors (and therefore total survey error) come at the price of time, resources, and money.  For example:
 
```

Comment:
Do we want to have one for each, or can we just add in a couple as examples.  These are already just examples, but I'm not sure all are needed to get the point across.

Reply:
I like talking through all of these.